assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],
+        "recipeFluids": ["0.25*Fluid:Water"],

```

Comment:
Yep just take it out :-)

You can make toast on a frying pan too, I suppose. Or roast it over an open flame on a simple metal grid holder thingie.